CHANGELOG.md

```diff
@@ -47,6 +48,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Changed
 
+- We merged `Citation information` and `Citation relations` into a singular tab. [#13618](https://github.com/JabRef/jabref/issues/13618)
```

Comment:
Please put only one entry in the changelog under this section `Changed`. Remove the others.